SwrveSDK/src/main/java/com/swrve/sdk/conversations/SwrveConversation.java

```diff
@@ -113,4 +119,15 @@ public ConversationPage getPageForControl(ControlBase control) {
     public SwrveConversationCampaign getCampaign() {
         return campaign;
     }
+
+    /**
+     * @return the message priority.
+     */
+    public int getPriority() {
+        return priority;
+    }
+
+    public void setPriority(int priority) {
```

Comment:
Also, the setter in the base class (setId/setName/setPages) should be removed for same reason.